calibr8/utils.py

```diff
@@ -111,10 +111,8 @@ def istensor(input:object) -> bool:
                     return True
     return False
 
-
 def plot_norm_band(ax, independent, mu, scale):
     """Helper function for plotting the 68, 90 and 95 % likelihood-bands of a Normal distribution.
-    
```

Comment:
These empty lines are part of the numpydoc style.

To edit them in the original commit try: `git rebase --interactive main`